### ISSUE TITLE
Bug: New (Italian? ) phone number that's valid, yet the library can't detect it as valid

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [googlei18n/libphonenumber#2123]
There is a phone number we've found, which seems to be valid (I can call it), yet the library says it's invalid:
+393901194828795

The check in this case (starting with "+") is as such:

```java
// input is String phoneNumberStr

                final PhoneNumberUtil phoneNumberUtil = PhoneNumberUtil.getInstance();
                try {
                    PhoneNumber internationalPhoneNumber = phoneNumberUtil.parse(phoneNumberStr, "");
                    if (internationalPhoneNumber != null && phoneNumberUtil.isValidNumber(internationalPhoneNumber))
                        return internationalPhoneNumber;
                } catch (Exception ex) {
                }
               return null
```

So if the returned value is null, it means the phone number is invalid.

It's probably a spam phone number. Seems to be from Italy.
I've tested on some websites, and they also claim it's invalid:
https://www.textmagic.com/free-tools/phone-validator
https://numverify.com/

I've also tried to call it (meaning to dial this phone number). On one device, it said the phone is blocked by the carrier. On another, it succeeded calling.

What is going on?
Could an invalid phone number still be call-able ?

**Comment by AndroidDeveloperLB** [comment]
We use latest version of the library, BTW:

    compile 'com.googlecode.libphonenumber:libphonenumber:8+'

**Comment by kkeshava** [comment]
@AndroidDeveloperLB,
Thank you for taking the time to improve libphonenumber! 
For your next metadata issue, we invite you to file issues in Google's new [Issue Tracker](http://issuetracker.google.com/issues/new?component=192347), which we're currently testing.

Italian phone numbers are max 11 digit numbers (both fixedline and mobile). The number provided by you seems to be a typo, i.e +39 3901194828795 is more than 11 digits. I think you have mentioned country code i.e 39 twice in your input which is wrong, try as +3901194828795. 

This is a valid fixedline numbers in Italy and we are already validating it as true, please see [demo](http://libphonenumber.appspot.com/phonenumberparser?number=%2B3901194828795).

**Comment by AndroidDeveloperLB** [comment]
@kkeshava Well, we've found that a user got a phone call from this number.
Also we've confirmed that it is possible to dial to it.
So the validation is incomplete here, because it is possible to have all these digits together in one phone number.
What do you mean by :

"For your next metadata issue, we invite you to file issues in Google's new Issue Tracker, which we're currently testing."
?

Anyway, please fix this issue.

**Comment by kkeshava** [comment]
@AndroidDeveloperLB,
We tried dialing the number +39 3901194828795 but we are getting recorded message "The number is not assigned. Please check the number again". Am I missing something here?

We need evidence to support any number, in this case especially as the length is different from what is given in official docs for Italy. Please see our [guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence). 
https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200006B0001PDFE.pdf
https://en.wikipedia.org/wiki/Telephone_numbers_in_Italy#Number_formats

**Comment by AndroidDeveloperLB** [comment]
@kkeshava Attached video with sound.
[VID_20180209_230842.zip](https://github.com/googlei18n/libphonenumber/files/1712432/VID_20180209_230842.zip)

Weird thing is that soon after I got a Hebrew (my main language) voice telling me they blocked me from calling this number of outside of the country. I've checked the settings of my account at the carrier, and it was fine : it should allow me to call outside of my country.

So this check might be faulty for some reason.

In any case, this doesn't explain how the user got a phone call from this number.

**Comment by AndroidDeveloperLB** [comment]
Maybe he was wrong and type the left digits twice, or something.

**Comment by AndroidDeveloperLB** [comment]
I tried to call without the 2 left digits, and still had the same music and voice. Maybe I should talk with my carrier, to see what's going on.

**Comment by AndroidDeveloperLB** [comment]
I will try to ask the user about this.

**Comment by kkeshava** [comment]
I'm closing as this issue is infeasible for us to fix at this point (we need more information). For your next metadata issue, we invite you to file issues in Google's new [Issue Tracker](https://b.corp.google.com/issues/new?component=192347&template=829703), which we're currently testing.

**Comment by AndroidDeveloperLB** [comment]
@kkeshava OK the user hasn't responded ever since I asked him. Maybe it's something weird he did by mistake.
